ci/nightly-test.sh

```diff
@@ -80,5 +80,6 @@ start_run_teardown "bond_its.sh"
 start_run_teardown "emergency_upgrade_test.sh"
 start_run_teardown "emergency_upgrade_test_balances.sh"
 start_run_teardown "sync_test.sh node=6 timeout=500"
+start_run_teardown "gov96.sh"
```

Comment:
We could give it a more descriptive name, e.g. `gov96_single_node_keeps_running.sh` or something?

Reply:
I tried to follow the example set by other `itst*` files. Also, I would have to rename config file to `gov96_single_node_keeps_running.config.toml` and it didn't seem right. I've updated the description of the test in the file so that its purpose is easily found. I hope it's OK with you if I don't rename it.